src/main/java/lotto/domain/GeneratedLotto.java

```diff
@@ -0,0 +1,51 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class GeneratedLotto {
+
+	private static final int LOTTO_NUMBER_MAX_COUNT = 6;
+
+	public List<Integer> lottoNumbers = new ArrayList<>();
+
+	public GeneratedLotto(){
+		Collections.shuffle(Lotto.defaultLottoNumbers);
+		for (int i = 0; i < LOTTO_NUMBER_MAX_COUNT; i++) {
+			lottoNumbers.add(Lotto.defaultLottoNumbers.get(i));
+		}
+	}
+
+	public GeneratedLotto(String[] inputNumber) throws IllegalArgumentException{
```

Comment:
```suggestion
	public GeneratedLotto(List<Integer> inputNumber) {
```

입력과 관련된 처리는 InputView에게 맡기세요
문자열을 입력 받았고 이 문자열을 로또에서 사용할 타입으로 변경해서 전달해주면 로또는 게임룰에만 집중할 수 있습니다.
문자열을 숫자로 바꾸고, trim 처리하는 등 입력값에 대한 처리를 신경쓰지 않아도 됩니다